galaxy/wrapper/bamCompare.xml

```diff
@@ -267,10 +281,12 @@
         <test expect_num_outputs="1">
             <param name="bamFile1" value="testA.bam" ftype="bam"/>
             <param name="bamFile2" value="testB.bam" ftype="bam"/>
-            <param name="showAdvancedOpt" value="yes"/>
+            <param name="method" value="own"/>
```

Comment:
should'nt this be in a conditional like above?

Reply:
Yes, these need to be fixed. I wrote in the matrix an issue with the tool. I can not test it ATM.